packages/snaps-utils/src/eval.ts

```diff
@@ -16,7 +14,7 @@ export async function evalBundle(bundlePath: string): Promise<null> {
   await validateFilePath(bundlePath);
 
   return new Promise((resolve, reject) => {
-    const worker = fork(WORKER_PATH, [bundlePath]);
+    const worker = fork(join(__dirname, 'eval-worker.js'), [bundlePath]);
```

Comment:
`__dirname` doesn't exist when running with ESM. I've simply inlined this, because it was only used in one place. The function itself isn't actually used.